staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/status_strategy.go

```diff
@@ -91,8 +91,12 @@ func (a statusStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.Obj
 
 		// validate x-kubernetes-validations rules
 		if celValidator, ok := a.customResourceStrategy.celValidators[v]; ok {
-			err, _ := celValidator.Validate(ctx, nil, a.customResourceStrategy.structuralSchemas[v], u.Object, cel.RuntimeCELCostBudget)
-			errs = append(errs, err...)
+			if hasBlockingErr(errs) {
+				errs = append(errs, field.Invalid(nil, nil, "Skipping validation rule due to OpenAPIv3 validation errors on fields accessible by validation rule"))
```

Comment:
"Skipping x-kubernetes-validations rules due to OpenAPIv3 required field, field length, array length, object length, or type errors in object" ?

+ @liggitt for any wordsmithing thoughts

Reply:
Do we need to list all the errors we considered in customer facing message?
I will update release note for it.

Reply:
"Skipping x-kubernetes-validations rules due to fundamental error against OpenAPIv3 schema"?

Reply:
Pushed change to update the err message to `Skipping x-kubernetes-validations rules due to OpenAPIv3 errors preventing the rules from safely executed`. Please let me know if something else is preferred^